### PR TITLE
STORM-1422 Remove broken example from the tutorial

### PR DESCRIPTION
**Pull request by sethgrid**
The tutorial code was broken giving an error. See http://stackoverflow.com/questions/34467446/apache-storm-classnotfoundexception-on-storm-starter. I did not alter example 2 because I have not verified that alteration.

### CODE REVIEW

File: examples/storm-starter/README.markdown [hunk 1]
```diff
@@ -104,8 +87,8 @@ Example filename of the uberjar:
 
 You can submit (run) a topology contained in this uberjar to Storm via the `storm` CLI tool:
 
-    # Example 1: Run the RollingTopWords in local mode (LocalCluster)
```
Comment (sethgrid) [inline examples/storm-starter/README.markdown]: `RollingTopWords` does not function either.

File: examples/storm-starter/README.markdown [hunk 2]
```diff
@@ -72,23 +72,6 @@ the Maven command to build and run storm-starter (see below), Maven will then be
 of Storm in this local Maven repository at `$HOME/.m2/repository`.
 
 
-## Running topologies with Maven
```
Comment (sethgrid) [inline examples/storm-starter/README.markdown]: None of the following examples work. See the linked SO question.

**Comment by vesense** [comment]
Thanks @sethgrid Can you file a JIRA for this?

**Comment by sethgrid** [comment]
@vesense after signing up for a JIRA account, I created the following. I hope it was set up properly: https://issues.apache.org/jira/browse/STORM-1422

**Comment by d2r** [comment]
> @vesense after signing up for a JIRA account, I created the following. I hope it was set up properly: https://issues.apache.org/jira/browse/STORM-1422

Looks good.  Can you edit the title of this pull request to include `STORM-1422` so they are linked?

**Comment by sethgrid** [comment]
@d2r updated title as per your request

File: examples/storm-starter/README.markdown (hunk 2 above)
Comment (d2r) [inline examples/storm-starter/README.markdown]: @ptgoetz, what do you think of this change to remove these instructions?

What is the use case for running topologies using maven?

**Comment by d2r** [comment]
@ptgoetz, what do you think of this change to remove these instructions? Asking because I see your name on that section of the README.

What is the use case for running topologies using maven?

It seems since storm-core is not in compile scope, it is not on the classpath for the exec plugin?

**Comment by ptgoetz** [comment]
@d2r I'm fine with updating the instructions. If I remember correctly (it's been a while ;) ), it was to make it very easy for new users to see a topology run in local mode.

+1 for the change.

**Comment by d2r** [comment]
Thanks @ptgoetz.

@sethgrid, I see also that this changes away from RollingTopWords. If RollingTopWords does not work properly, could you create a new Jira issue to fix that separately?  We do not have to fix it with this change.

+1

**Pull request merged**